modules/frontend/pipeline/trace_query_filter_test.go

```diff
@@ -0,0 +1 @@
+package pipeline
```

Comment:
i would look at this test:

https://github.com/grafana/tempo/blob/c300244823ca1b068535ce3207aeb7edff1b52c7/modules/frontend/pipeline/async_handler_multitenant_test.go#L112

for an example on how to create and exercise the middleware